### ISSUE TITLE
Clarify formula for membership of Steering committee

### ISSUE THREAD

**Issue by sebgoa** [knative/community#33]
Currently the SC is populated based on:

> Seats on the steering committee are allocated based upon contribution to the project by an organization. No final decision has been made on the exact formula.

I think it is time to clarify this formula. Until the formula is clearly spelled out this does not feel like an open governance to me and with Google's majority of 4 seats, this feels too much like a Google project. This is starting to cause a perception issue in the large k8s/cloud-native community which I strongly believe needs to be addressed.

I would recommend to look at the way kubernetes setup the bootstrap committee and subsequently held elections.

**Comment by rgregg** [comment]
I've personally struggled with how we can fairly measure contribution to the project. Not all contributions come in the form of measurable data. There's certainly things like LOC, PRs, issues opened, issues resolved, issues worked on, leadership, presentations, code reviews, $'s spent, etc. Trying to find an exact formula that we can plug data into and produce a meaningful output hasn't been a priority, although maybe it should be. 

I'd be interested to know if there are examples we can reference or other formulas that have been used in other projects to measure the contributions of the community. I'd rather not invent our own formula for Knative if we can avoid it.

In the past we've referenced the [CNCF DevStats page for Knative](https://knative.teststats.cncf.io/d/5/companies-table?orgId=1), to gather some insights into contributions to the project, although it feels like a bit of a black box in terms of what that measures. If we used that metric alone, the SC make-up would be even more skewed, so we originally balanced it based on orders of magnitude of contributions across the companies listed.

I'm open to suggestions for how we can measure contributions objectively to create a formula, but I don't see us reconsidering the organization style membership of the Steering committee right now.
